db/sysinfo.go

```diff
@@ -92,6 +93,7 @@ func TopSysinfoFields() ([]TopSysinfo, error) {
 		SELECT 1 as order, * FROM top_field('Distro')
 		UNION SELECT 2 as order, * FROM top_field('DeWm')
 		UNION SELECT 3 as order, * FROM top_field('DisplayProtocol')
+		UNION SELECT 3 as order, * FROM top_field('Shell')
```

Comment:
make it SELECT 4 and change all the below for that

Reply:
Good catch, I didn't even notice.

I did literally just copy-paste without thinking.